examples/bootstrap4/app.py

```diff
@@ -89,6 +88,36 @@ class ContactForm(FlaskForm):
     im_accounts = FieldList(FormField(IMForm), min_entries=2)
 
 
+class BootswatchForm(FlaskForm):
+    """Form to test Bootswatch."""
+    #DO NOT EDIT! Use list-bootswatch.py te generate the Radiofield below.
+    render = RadioField(default='default',
```

Comment:
`theme` or `theme_name` will be a better name than `render`.

Reply:
fixed